src/index.js

```diff
@@ -1,4 +1,5 @@
 const countriesData = require("./country-cities.json")
+const CurrencyExplorer = require('currency-explorer');
```

Comment:
this constant should be camel case
also we only need `const { currencies } = require('currency-explorer');`

also kindly lazy load this inside `fetchCurrencies` as it's only used there